frontend/pages/g/_groupSlug/r/create/plaintext.vue

```diff
@@ -0,0 +1,103 @@
+<template>
+  <div>
+    <v-form ref="domUrlForm" @submit.prevent="createRecipe">
+      <div>
+        <v-card-title class="headline">{{ $t('recipe.create-recipe-from-text') }}</v-card-title>
+        <v-card-text>
+          <p>{{ $t('recipe.create-recipe-from-text-description') }}</p>
+          <v-container class="pa-0">
+            <v-row>
+              <v-col cols="12">
+                <v-textarea
+                  v-model="recipeText"
+                  :label="$t('recipe.enter-recipe-text')"
+                  :placeholder="$t('recipe.recipe-text-placeholder')"
```

Comment:
For the placeholder, I think a short example would work better. Something like: "The best ham and cheese sandwich you'll ever make. Made with ham, cheese, bread, and mayo."

Alternatively we could do without the placeholder, I think the description above should be enough.